content/js/record.md

```diff
@@ -270,40 +275,15 @@ let note2 = new Note({
   content: 'This is second section'
 });
 note1.nextPage = skygear.Reference(note2);
+skygear.publicDB.save([note2, note1]);
+// success, ordering in the batch save array really matters
```

Comment:
- no "really" here
- need some descriptions saying that you are showing how dependent records should be saved